modules/ove-about.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * overview/ove-overview.adoc
+
+:_module-type: CONCEPT
+[id="ove-about_{context}"]
+= About {product-title}
+
+While all editions of Red Hat OpenShift include link:https://www.redhat.com/en/technologies/cloud-computing/openshift/virtualization[{virt}], {product-title} is a cost-effective, virtualization-only option that does not include container-based or cloud-native features for applications. If you migrate from a traditional virtualization platform to {product-title}, you can add containerization and modernization features later by switching to a different edition of OpenShift.
+
+[NOTE]
+====
+All editions of OpenShift include container-based infrastructure workload hosting.
+====
+
+To simplify migration and VM management, {product-title} supports the following tools and capabilities:
+
+* Red Hat Ansible Automation Platform - Enables automation at scale, streamlining tasks such as VM provisioning, migration, monitoring, and management.
```

Comment:
Unless this styling is common in the CNV docs, I suggest using a [definition list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) to better align with complex list treatment in the ISG.

Reply:
I had done it this way to match the OPP docs, but I agree and will change this.